tests/test_time.cc

```diff
@@ -27,6 +30,15 @@ namespace test {
 
 //-----------------------------------------------------------------------------
 
+void assertTypeExpansion(const std::string& name, std::vector<std::string> values,
+                         const std::vector<std::string>& expected) {
+    std::cout << "comparing " << values << " with " << expected;
```

Comment:
Do we want this. Or even eckit::Log by default in the output?
Surely we should be quiet unless it fails?